__tests__/Port.test.js

```diff
@@ -0,0 +1,38 @@
+/* globals describe it expect */
```

Comment:
do you need this comment? you don't have it in your Itinerary test file and it doesn't cause any problems when I delete on my end